src/py/flwr/common/serde_test.py

```diff
@@ -425,15 +425,19 @@ def test_metadata_serialization_deserialization() -> None:
     """Test serialization and deserialization of Metadata."""
     # Prepare
     maker = RecordMaker()
-    original = maker.metadata()
+    metadata = maker.metadata()
+
+    # A Metadata obj is fully defined when it's used by a Message
+    # The Message sets the metadata's .created_at
+    mssg = Message(metadata=metadata, content=maker.recordset(1, 0, 0))
```

Comment:
A `Metadata` object is not fully defined if it's not used to construct a `Message`. that's because the [`Message` constructor](https://github.com/adap/flower/blob/354af13409ba98f37dcd7c4dad702c76ac3b6d37/src/py/flwr/common/message.py#L222) sets the metadata`.created_at`. Because of this reason, this test is identical to the next one so I vote for its removal. @chongshenng @danieljanes

Reply:
Good point. +1 for this.

Reply:
I've removed this.